proto/topsql_agent.proto

```diff
@@ -13,8 +13,11 @@ option (gogoproto.unmarshaler_all) = true;
 
 // TopSQLAgent is the persistent agent service for TopSQL records
 service TopSQLAgent {
+    // DEPRECATED: use ReportTopSQLRecords instead.
     // ReportCPUTimeRecords is called periodically (e.g. per minute) to save the in-memory TopSQL records
     rpc ReportCPUTimeRecords(stream CPUTimeRecord) returns (EmptyResponse) {}
```

Comment:
Seems that we will never call this API more. If so, then maybe you can remove this interface.

Reply:
OK, I have deleted it and tested on TiDB and NGM.